vck/src/commonMain/kotlin/at/asitplus/wallet/lib/agent/Validator.kt

```diff
@@ -489,4 +529,119 @@ class Validator(
         return Verifier.VerifyCredentialResult.SuccessIso(it)
     }
 
+
+    /**
+     * Validate a status list jwt within a relevant context.
+     */
+    fun validateStatusListJwt(
+        it: String,
+        statusListInfo: StatusListInfo,
+        resolvedAt: Instant?,
+    ): KmmResult<Unit> = at.asitplus.catching {
+        Napier.d("setRevocationStatusListCwt: Loading $it")
+        val payload = validateStatusListJwtIntegrity(it).getOrThrow()
+        StatusListTokenValidator.validateStatusListTokenPayloadClaims(
+            statusListTokenPayload = payload,
+            statusListInfo = statusListInfo,
+            statusListTokenResolvedAt = resolvedAt,
+            isInstantInThePast = {
+                it < clock.now()
+            },
+        ).getOrThrow()
+        Napier.d("Token is valid")
+    }
+
+    /**
+     * Validate a status list cwt within a relevant context.
+     */
+    fun validateStatusListCwt(
```

Comment:
This method is never called

Reply:
fixed in 93ce860696aa4077acb9a053a8b3eac9bee2733b